src/panel/direction/RouteResult.jsx

```diff
@@ -108,9 +108,9 @@ export default class RouteResult extends React.Component {
                   </div>
                   <PlaceholderText length={10} className="itinerary_leg_via_details"/>
                 </div>
-                <div className="itinerary_leg_info">
-                  <PlaceholderText length={5} className="itinerary_leg_duration" />
-                  <PlaceholderText length={7} className="itinerary_leg_distance" />
+                <div>
```

Comment:
Thanks for giving some :heart: to the placeholders :)

Reply:
It's not perfect at all, I just clean up deprecated classes. I will try to do another pass once the design is done